src/Reducer.h

```diff
@@ -229,7 +229,8 @@ bool bLZCHasHW=false;
 
       // The condition (abs(v_) | abs(x_)) <= THRESH protects against
       // overflow
-      below_threshold = (labs(v_) | labs(x_)) <= THRESH ? 1 : 0;
+      // must use llabs() as labs() is undefined for too large numbers
+      below_threshold = (llabs(v_) | llabs(x_)) <= THRESH ? 1 : 0;
```

Comment:
technically, `llabs()` is supposed to be declared in `<cstdlib>`. https://en.cppreference.com/w/cpp/numeric/math/abs
Perhaps it ends up being included indirectly

Reply:
ya unsure, the original code was abs() and depended on the overloads. maybe the compiler wasn't happy with that for the hwvdf changes, but that got switched to the wrong explicit form.